fineract-provider/src/main/java/org/apache/fineract/commands/domain/CommandSource.java

```diff
@@ -129,8 +130,11 @@ public void markAsRejected(final AppUser checker, final DateTime checkedOnDate){
         this.checkedOnDate = checkedOnDate.toDate();
         this.processingResult = CommandProcessingResultType.REJECTED.getValue();
     }
+    
+
+   
 
-    public void updateResourceId(final Long resourceId) {
+	public void updateResourceId(final Long resourceId) {
```

Comment:
This file should not be part of pull request as it is having only formatting issue